gmailfilter/resource_gmailfilter_label.go

```diff
@@ -14,7 +14,25 @@ func resourceGmailfilterLabel() *schema.Resource {
 		Update: resourceGmailfilterLabelUpdate,
 		Delete: resourceGmailfilterLabelDelete,
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				config := meta.(*Config)
+				name := d.Id()
+
+				res, err := config.gmailService.Users.Labels.List(gmailUser).Do()
+				if err != nil {
+					return nil, handleNotFoundError(err, d, "Label")
+				}
+
+				for _, l := range res.Labels {
+					if l.Name == name {
+						d.SetId(l.Id)
+						d.Set("name", d.Id())
```

Comment:
This line is unnecessary.
State will be refreshed with the label's ID during import.

Reply:
I see. Thanks!
Fix in 9bfbcb5